cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -115,15 +115,24 @@ jobs:
       - uses: swatinem/rust-cache@v2
       - name: Install cargo-dist
         run: ${{ matrix.install_dist }}
+      - name: Install dependencies
+        run: |
+          ${{ matrix.packages_install }}
       - id: cargo-dist
+        name: Build
         # We force bash here just because github makes it really hard to get values up
         # to "real" actions without writing to env-vars, and writing to env-vars has
         # inconsistent syntax between shell and powershell. cargo-dist and jq work fine
         # in powershell.
         shell: bash
         run: |
+          CARGO="cargo"
+          if [[ -f "Brewfile" ]]; then
+            CARGO="brew bundle exec -- cargo"
+          fi
+
```

Comment:
Oh yeah, didn't we agree it should be possible/ideal for these expressions to be something that `cargo dist build` does for you..?

Reply:
I think so, yeah. I don't mind keeping this like this for now though - a new subcommand that handles this would be a nice stretch goal, especially since we *may* have an external crate to use for that. It's IMO not critical until we have a second CI backend.

Reply:
ok i'm fine with landing it like this, if we don't like it we can clean it up in followups